README.md

```diff
@@ -49,35 +49,25 @@ $ cd apisix-dashboard
 
 The `manager-api` is used to provide APIs for Dashboard, just like a bridge between the Apache APISIX and the Dashboard. Here are the steps to build it manually:
 
-1. We need `MySQL/Golang` to be preinstalled.
+1. We need `Go` 1.13+, `ETCD` 3.4+ and `Lua` 5.1+ to be preinstalled.
```

Comment:
Why we need lua?

Reply:
> But I think we can build it in development phase, no need build it at installing

agree +1

Reply:
@moonming @ShiningRush @membphis 
we need Lua to generate jsonschema and to generate lua code for plugin orchestration.

Reply:
@nic-chen 

take a look at this project: https://github.com/yuin/gopher-lua . 
I think it should be a better way.

Reply:
And why we need to install etcd first?

Reply:
manager api depend on ETCD, so we need a etcd instance before run manager api.

Reply:
Is this the same etcd used by Apache APISIX? 
Here are the compilation steps. It can be compiled without etcd, right?

Reply:
> Is this the same etcd used by Apache APISIX?
Yes
> Here are the compilation steps. It can be compiled without etcd, right?
You are right, we can compile it without etcd

Reply:
@membphis 
we could take a try later.

Reply:
@moonming @ShiningRush  I have updated the doc about ETCD. please take a look.